docs/latex/Inhalt/03_Inhalt/06_programming_principles.tex

```diff
@@ -1,6 +1,22 @@
 \chapter{Programming Principles}
+Programming Principles oder zu deutsch Programmierprinzipien sind eine Verallgemeinerung wiederkehrender Erkenntnisse in der Softwareentwicklung und liefern Entwicklern Vorgaben für einen bestimmten Programmierstil. Speziell für die objektorientierte Programmierung existieren die objektorientierten Prinzipien, welche als Idealvorstellung für \enquote{guten} objektorientierten Code dienen. Einige dieser Prinzipien werden in den nachfolgenden Abschnitten vorgestellt und deren Anwendung im vorliegenden Projekt untersucht.
 
 \section{SOLID}
+Die SOLID-Prinzipien sind sind eine Sammlung von Regeln mit dem Ziel wartbare und erweiterbare Software mit einer langlebigen Codebasis zu schreiben. SOLID steht hierbei für Single Responsibility Principle, Open/Closed Principle, Liskov Substitution Principle, Interface Segregation Principle und Dependency Inversion Principle.
```

Comment:
"Die SOLID-Prinzipien sind sind eine Sammlung von Regeln mit dem Ziel wartbare und erweiterbare Software mit einer langlebigen Codebasis zu schreiben" => "mit einer langlebigen Codebasis" klingt komisch

Reply:
"SOLID steht hierbei für " => "Jeder Buchstabe in SOLID steht für den Namen eines einzelnen Prinzips. Im Folgenden wir auf jedes der Prinzipien im Detail eingegangen"

Reply:
Die langlebige Codebasis kommt von seinen Folien. Das andere habe ich umgesetzt.